Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y gcc make
+
+COPY . /app
```

Comment:
If someone made changes to their solution code and wanted to try running it again, wouldn't they have to rebuild the image to copy over the updated file? It'd be nice if they didn't have to do that.

Reply:
Done, they can now use the bash shell in the container to compile/run their solutions.